typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -223,19 +245,23 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig<
     rarichain: true,
     real: true,
     redstone: true,
+    rivalz: true,
```

Comment:
rivalz wasn't in the PR description but assuming it is part of the new batch

Reply:
good catch - i updated changeset but not description